docs/src/main/sphinx/connector/jdbc-common-configurations.fragment

```diff
@@ -33,3 +33,10 @@ connector:
       Do not change this setting from the default. Non-default values may
       negatively impact performance.
     - 1000
+  * - ``join-pushdown.enabled``
+    - Enable join pushdown. Join pushdown is disabled by default as this is the
+      safer option. Pushing down a join which substantially increases the row
+      count vs sizes of left and right table separately, may incur huge cost
+      both in terms of performance and money due to an increased network
+      traffic.
```

Comment:
Like the `write.batch-size` property above, we probably don't need to give much more detail than "can negatively impact performance" or similar.

Reply:
Fair point. The information isn't actionable anyway since the only option is "yes" or "no". 👍